C/casacore/build_tarballs.jl

```diff
@@ -13,18 +13,25 @@ sources = [
 
 # Bash recipe for building across all platforms
 script = raw"""
-OPENBLAS=(-lopenblas64_)
-
 cd $WORKSPACE/srcdir/casacore
 
+if [[ "${nbits}" == 64 ]] && [[ "${target}" != aarch64* ]]; then
```

Comment:
This isn't right.  With latest versions of OpenBLAS we do ILP64 also for aarch64.  Related, I'd recommend requiring Julia v1.7 below, because in Julia v1.6 we still used to have an OpenBLAS which didn't use ILP64 for aarch64.

Reply:
Alternatively, use `OpenBLAS32_jll` and you'll have `libopenblas.${dlext}` everywhere, which is maybe easier than renaming all symbols...

Reply:
Is there a downside to just using OpenBLAS32_jll?

Reply:
You can index arrays much smaller than using `Int64` indices, but most packages don't really care.